src/components/manage/Sidebar/Sidebar.jsx

```diff
@@ -56,14 +58,48 @@ class Sidebar extends Component {
   constructor(props) {
     super(props);
     this.onToggleExpanded = this.onToggleExpanded.bind(this);
+    this.onToggleFullSize = this.onToggleFullSize.bind(this);
     this.onTabChange = this.onTabChange.bind(this);
     this.state = {
       expanded: cookie.load('sidebar_expanded') !== 'false',
+      size: 0,
+      showFull: true,
+      showFullToolbarExpanded: true,
     };
   }
 
+  /**
+   * on component did mount check for toolbar expanded
+   */
+  componentDidMount() {
+    this.chechForToolbarExpanded();
+  }
+
+  /**
+   * check for toolbar expanded
+   * set listener for expand/minimize to adjust width of sidenav
+   * @returns {undefined}
+   */
+  chechForToolbarExpanded() {
```

Comment:
no need to checkForToolbarExpanded if you keep track of the state of the toolbar